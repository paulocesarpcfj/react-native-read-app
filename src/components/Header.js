import React from 'react';
import { Platform, View, Text } from 'react-native';

const headerStyles = {
    fontSize: 20,
    padding: 15,
    paddingTop: (Platform.OS === 'ios') ? 25 : 15,
    textAlign: 'center',
    backgroundColor: '#F2F2F2',
};

export default () => (
    <View style={{ borderBottomWidth: 1, borderBottomColor: '#CCC' }}>
        <Text style={headerStyles}>
            Read
        </Text>
    </View>
);
