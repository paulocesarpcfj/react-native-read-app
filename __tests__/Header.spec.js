import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import Header from '../src/components/Header';

it('renders correctly', () => {
    const header = renderer.create(
        <Header />
    ).toJSON();
    expect(header).toMatchSnapshot();
});
