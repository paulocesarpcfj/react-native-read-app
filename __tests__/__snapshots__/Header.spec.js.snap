// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "#CCC",
      "borderBottomWidth": 1,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "backgroundColor": "#F2F2F2",
        "fontSize": 20,
        "padding": 15,
        "paddingTop": 25,
        "textAlign": "center",
      }
    }
  >
    Read
  </Text>
</View>
`;
